packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -127,6 +127,12 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     }
     if (activeTool !== prevProps.activeTool) {
       this.updateSelectionValidators();
+      if (activeTool === ToolType.DEFAULT) {
+        this.setState({
+          linkInProgress: undefined,
+          selectedIds: new Set<string>(),
+        });
```

Comment:
You could probably put this in a `reset()` method. I just noticed in `handleAllLinksDeleted` we're setting `linkInProgress:undefined` but we're not resetting `selectedIds`, and we probably should.